File: app/src/main/java/com/sorcererxw/matthiasheiderichphotography/ui/others/MHItemAnimInterpolator.java
package com.sorcererxw.matthiasheiderichphotography.ui.others;


import android.view.animation.Interpolator;

/**
 * @description:
 * @author: Sorcerer
 * @date: 2016/12/15
 */

public class MHItemAnimInterpolator implements Interpolator {

    @Override
    public float getInterpolation(float input) {
        return (float) Math.pow(input, 4);
    }
}

File: app/src/main/java/com/sorcererxw/matthiasheiderichphotography/ui/others/MHItemAnimInterpolator.kt
package com.sorcererxw.matthiasheiderichphotography.ui.others


import android.view.animation.Interpolator

/**
 * @description:
 * @author: Sorcerer
 * @date: 2016/12/15
 */

class MHItemAnimInterpolator : Interpolator {
    override fun getInterpolation(input: Float): Float {
        return Math.pow(input.toDouble(), 4.0).toFloat()
    }
}
